LoginApp(new)/LoginWindow/main.cpp

```diff
@@ -0,0 +1,11 @@
+#include "mainwindow.h"
+#include <QApplication>
+
+int main(int argc, char *argv[])
+{
+    QApplication a(argc, argv);
+    MainWindow w;
+    w.show();

```

Comment:
According to code convention variable names have to describe their purposes. "Do not use short forms"